Felienne/hedy
# [PERFORMANCE] Makes dropdown menu touch friendly

**Description**
In this PR we switch from the `hover` state as feature to show the dropdown to the `click` state. This has several advantages, but the biggest one is that it makes our website more touch friendly. With the current hover state it is hard for touchscreen users to get the language selector or profile options dropdown to shown. With this implementation we remove the hover dropdown and only show the menu on click, keeping it shown until clicked outside of the dropdown menu.

**Fixes**
This PR is a starting point for #2925, with the goal to make the menubar overall beter accessible.
For now this only enhances the touchscreen experience, but in a different PR we can built upon this.

**How to test**
Use a browser that can mimic a touchscreen device, such as Firefox -> developers tools. Verify that the dropdown menu from either the language selector or the profile options now works as expected: dropping down when clicked. Also verify that the menu is only hidden iff clicked outside the dropdown itself (or clicking the button again).
